src/System.Net.Sockets.Legacy/ref/System.Net.Sockets.Legacy.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the http://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+
+namespace System.Net.Sockets
+{
+   
+    public static class Legacy
```

Comment:
We should have a better name for this class. The general pattern is <something>Extensions for static classes with extensions.